ietf-te-path-computation.yang

```diff
@@ -236,7 +249,7 @@ module ietf-te-path-computation {
       }
       uses te-types:generic-path-constraints;
       uses te-types:generic-path-optimization;
-      
+      uses returned-metrics-info;
```

Comment:
@efralaz : please check if there is a way to preclude the metric-value being specified in the request

Reply:
Not an economic way. It's just possible to create two different groupings, one for the request (without the returned values) and one for the reply (with the returned values, mandatory). 
As in many other places of the proposed standard (including te-tunnel and similia) optional fields are widely used for the purpose of defining one single grouping with multiple usage, I suggest to keep the same approach and specify in the description that the returned value is optional and must be not present (and if present it shall be ignored) in the request and must be present in the reply.